pkgs/by-name/mi/mistral-rs/package.nix

```diff
@@ -178,6 +184,28 @@ rustPlatform.buildRustPackage rec {
     "--skip=util::tests::test_parse_image_url"
   ];
 
+  postFixup = lib.optionalString cudaSupport (
+    let
+      patchelfCommand = binaryName: ''
+        patchelf \
+          --add-rpath ${
```

Comment:
Can we use `autoPatchelfHook` with `runtimeDependencies` (Since there are only this two executables)?

https://nixos.org/manual/nixpkgs/unstable/#setup-hook-autopatchelfhook

Reply:
Thanks for the tip. I went and try it and it doesn't work weirdly...
I'm pushing the changes.

Reply:
The `ldd` output is the same, but I guess that the `rpath` is not about that.

Reply:
![image](https://github.com/user-attachments/assets/88d2aaeb-b6f2-4eee-9221-2e4c30eb7e5b)

This is the result of `patchelf --print-rpath` for the two variants.